core/src/main/java/nl/nn/adapterframework/jdbc/dbms/translator/MssqlTranslator.java

```diff
@@ -0,0 +1,35 @@
+package nl.nn.adapterframework.jdbc.dbms.translator;
+
+public class MssqlTranslator extends ITranslator {
+	public MssqlTranslator(ITranslator target) {
+		super(target);
+	}
+
+	public MssqlTranslator() {
+		super();
+	}
+
+	@Override
+	protected void populateMaps() {
+		regexes.put("NEXTVAL", toPattern("NEXT VALUE FOR (\\w+)"));
+		replacements.put("NEXTVAL", "(NEXT VALUE FOR $1)");
+
+		regexes.put("CURRVAL", toPattern("SELECT CURRENT_VALUE FROM SYS.SEQUENCES WHERE NAME\\s+=\\s+'(\\w+)'"));
+		replacements.put("CURRVAL", "(SELECT current_value FROM sys.sequences WHERE name = '$1')");
```

Comment:
This will work only when the query searched has exactly this form

Reply:
I know, currently these regex strings are more of a placeholder. I'm testing and improving them as i go along, one by one.